### PR TITLE
Accessibility improvements

### PR DESCRIPTION
**Pull request by schaerta**
Some improvements after user tests.
Note: The linter complained when I tried to use an invisible link that just serves as a button. I also googled around and it just seems like a bad way of solving the problem. So I added an invisible heading instead, that should make the signup easier to find.
If you can think of anything else to add to improve a11y, please let me know!

### CODE REVIEW

File: src/components/TickerToSignup/SignupButton/index.js [hunk 1]
```diff
@@ -10,7 +10,11 @@ export const SignUpButton = ({ children, className }) => {
   return (
     <>
       <div className={s.signUpButton}>
-        <Button className={className} onClick={() => setOverlayOpen(true)}>
+        <Button
+          className={className}
+          ariaLabel="Anmelden"
```
Comment (Valioesi) [inline src/components/TickerToSignup/SignupButton/index.js:15]: Heißt es nicht aria-label?

**Review by Valioesi — CHANGES_REQUESTED** [review]
ariaLabel instead of aria-label

File: src/components/TickerToSignup/SignupButton/index.js (hunk 1 above)
Comment (schaerta) [reply src/components/TickerToSignup/SignupButton/index.js]: Oh, da hast du wohl recht. Dachte, es wäre camel case, wie sonst auch bei JSX. Habe es angepasst.

**Review by nmauersberg — APPROVED** [review]
Super cool, thank you! <3

**Review by Valioesi — APPROVED** [review]
Thank you! :)

**Pull request merged**